### PR TITLE
chore: add config.yml for welcome bot

### LINKED ISSUE #42: [Feat]: Create config.yml for Welcome Bot
## Description
As a first time contributor,
I need a bot to help guide my first contribution,
so that I can follow this org's contribution guidelines.

## Mocks
welcomebot

## Acceptance Criteria
- [ ] newIssueWelcomeComment is complete with the correct links to this repo.
- [ ] newPRWelcomeComment is complete with the links to the project's stream or topic and the repo's Code of Conduct.
- [ ] firstPRMergeComment is complete.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by nlok5923**
### Description

this pull request add config.yml file for welcome bot

Fixes #42 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- config.yml file for welcome bot

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: .github/workflows/config.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉for opening your very first issue in this project. AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md).🙌
```
Comment (jalajcodes) [inline .github/workflows/config.yml:10]: We have a Code of conduct file in this repo too, maybe link that instead.

File: .github/workflows/config.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  You may submit a PR if you like. If you want to report a bug🐞 please follow our [ISSUE_TEMPLATE.md](https://github.com/anitab-org/.github/tree/main/.github/ISSUE_TEMPLATE). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
```
Comment (jalajcodes) [inline .github/workflows/config.yml:11]: Same here

File: .github/workflows/config.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉 for opening your first pull request.✨ AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/anitab-org/open-source-programs-backend/blob/develop/.github/CONTRIBUTING.md).🙌.We will get back to you as soon as we can.😄
```
Comment (jalajcodes) [inline .github/workflows/config.yml:24]: Here too 😅

**Review by jalajcodes — CHANGES_REQUESTED** [review]

File: .github/workflows/config.yml (hunk 3 above)
Comment (nlok5923) [reply .github/workflows/config.yml]: addressed

File: .github/workflows/config.yml (hunk 2 above)
Comment (nlok5923) [reply .github/workflows/config.yml]: addressed

File: .github/workflows/config.yml (hunk 1 above)
Comment (nlok5923) [reply .github/workflows/config.yml]: addressed

**Comment by nlok5923** [comment]
@annabauza please review this pr.

File: .github/workflows/config.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉for opening your very first issue in this project. AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/stem-diverse-tv-cms/blob/master/.github/CODE_OF_CONDUCT.md).🙌
+  You may submit a PR if you like. If you want to report a bug🐞 please follow our [ISSUE_TEMPLATE.md](https://github.com/anitab-org/stem-diverse-tv-cms/tree/master/.github/ISSUE_TEMPLATE). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
```
Comment (Rahulm2310) [inline .github/workflows/config.yml:11]: As there is already an issue open for deleting the ISSUE_TEMPLATE (#46), you can update the link to the issue template under [.github](https://github.com/anitab-org/.github) repo.

Comment (nlok5923) [reply .github/workflows/config.yml:11]: yeah sure thanks @Rahulm2310

**Review by Rahulm2310 — APPROVED** [review]
LGTM 💯

**Review by jalajcodes — APPROVED** [review]

**Review by annabauza — APPROVED** [review]

**Pull request merged**